documentation/modules/con-configuring-user-operator.adoc

```diff
@@ -9,7 +9,7 @@ User Operator deployment can be configured using additional options inside the `
 The following properties are supported:
 
 `watchedNamespace`::
-The Kubernetes namespace in which the topic operator watches for `KafkaUsers`.
+The Kubernetes namespace in which the user operator watches for `KafkaUsers`.
```

Comment:
Why is this here?

Reply:
It's an unrelated typo.

Reply:
@laidan6000 which should not be in unrelated PRs :-) ... anyway it's ok for this time! :P